provider/provider_test.go

```diff
@@ -0,0 +1,195 @@
+package github_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/pulumi/providertest/replay"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	pulumirpc "github.com/pulumi/pulumi/sdk/v3/proto/go"
+
+	github "github.com/pulumi/pulumi-github/provider/v6"
+	"github.com/pulumi/pulumi-github/provider/v6/pkg/version"
+)
+
+const repro586 = `[
```

Comment:
Could be adventurous and actually use provdertest upgrade tests here :) But this works.

Reply:
I'll do that next time.